.circleci/config.yml

```diff
@@ -35,9 +68,6 @@ job-references:
       git submodule update --init --recursive
 
   php_job: &php_job
-    environment:
```

Comment:
Turns out these env variables were ignored. As when this block gets included in a job, job's environment section overwrites (not merges) with these. That is why I moved to use of pipeline parameters above instead.